finetuner-workflow/finetune-workflow.yaml

```diff
@@ -582,31 +702,42 @@ spec:
                             - "{{workflow.parameters.region}}"
             containers:
               - name: kfserving-container
-                image: "{{workflow.parameters.finetuner_image}}:{{workflow.parameters.finetuner_tag}}"
+                image: "{{workflow.parameters.inference_image}}:{{workflow.parameters.inference_tag}}"
                 imagePullPolicy: IfNotPresent
-                command: ["/usr/bin/bash", "-c"]
-                args: ["/usr/bin/python3 /app/inference.py --model=${INFERENCE_MODEL} --port=${INFERENCE_PORT}"]
+                command: ["/bin/bash", "-c"]
+                args:
+                  - |
+                    python -m vllm.entrypoints.openai.api_server \
+                      --model ${INFERENCE_MODEL_PATH} \
+                      --model-loader-extra-config "{\"tensorizer_uri\": \"${INFERENCE_MODEL_URI}\"}" \
+                      --load-format tensorizer \
+                      --port ${INFERENCE_PORT} \
+                      --dtype ${INFERENCE_PRECISION}
                 env:
                   - name: INFERENCE_PORT
                     value: "80"
-                  - name: INFERENCE_MODEL
+                  - name: INFERENCE_MODEL_URI
+                    value: /mnt/pvc/{{ inputs.parameters.model_uri }}
+                  - name: INFERENCE_MODEL_PATH
                     value: /mnt/pvc/{{ inputs.parameters.model_path }}
                   - name: STORAGE_URI # KFServing mounts the PVC at /mnt/pvc/
                     value: pvc://{{ workflow.parameters.pvc }}/
+                  - name: INFERENCE_PRECISION
+                    value: {{ workflow.parameters.inference_precision }}
                 ports:
                   - protocol: TCP
                     containerPort: 80
                 livenessProbe:
                   httpGet:
-                    path: /
+                    path: /health
                     port: 80
-                  initialDelaySeconds: 300
+                  initialDelaySeconds: 60
```

Comment:
Maybe bump these back up a little higher, since it timing out too early would cause more issues than taking a little too long to time out in the error case.